packages/components/messages/src/additional-info-message/additional-info-message.tsx

```diff
@@ -0,0 +1,37 @@
+import { ReactNode } from 'react';
+import Text from '@commercetools-uikit/text';
+
+export type TIntlMessageDescriptor = {
+  id: string;
+  description?: string | object;
+  defaultMessage: string;
+  values?: Record<string, ReactNode>;
+};
+
+export type TAdditionalInfoMessageProps = {
+  id?: string;
+  message: string | ReactNode | TIntlMessageDescriptor;
```

Comment:
This should be renamed to `intlMessage` right?

Reply:
I think it's better to keep it like this.
We use the `intlMessage` name when the prop is expected to receive an i18n message: a message object description.

In this case, the prop is polymorphic as it can receive different types (an already internationalized message string or even an element, for instance) so keeping the current name looks more appropirate to me.